### PR TITLE
EIP1-1990: EIP1-2593: Implement not found scenario for certificate summary endpoint

### PR DESCRIPTION
**Pull request by adacayi**
This PR is to implement the certificate not found scenario for the certificate summary endpoint

### CODE REVIEW

File: src/test/kotlin/uk/gov/dluhc/printapi/config/ApiRequestErrorAttributesTest.kt [hunk 1]
```diff
@@ -0,0 +1,81 @@
...
+        val actual = apiRequestErrorAttributes.getErrorResponse(webRequest)
+
+        // Then
+        assertThat(actual)
+            .hasTimestampNotBefore(earliestExpectedTimeStamp)
+            .hasStatus(400)
+            .hasError("Bad Request")
+            .hasMessage("Validation failed for object='testBean'. Error count: 1")
+            .hasValidationError("Error on field 'aField': rejected value [a], should be greater than 2 chars in length")
+    }
```
Comment (Neil-Massey) [inline src/test/kotlin/uk/gov/dluhc/printapi/config/ApiRequestErrorAttributesTest.kt:75]: Do we need a test for the `Required request body is missing` error message sanitization flow?

File: src/test/kotlin/uk/gov/dluhc/printapi/rest/GetCertificateSummaryByApplicationIdIntegrationTest.kt [hunk 2]
```diff
@@ -81,6 +81,24 @@ internal class GetCertificateSummaryByApplicationIdIntegrationTest : Integration
...
+
+        // When
+        val response = webTestClient.get()
+            .uri(URI_TEMPLATE, ERO_ID, APPLICATION_ID)
+            .bearerToken(getBearerToken(eroId = ERO_ID, groups = listOf("ero-$ERO_ID", "ero-vc-admin-$ERO_ID")))
+            .contentType(MediaType.APPLICATION_JSON)
+            .exchange()
+
+        // Then
+        response.expectStatus().isNotFound
```
Comment (Neil-Massey) [inline src/test/kotlin/uk/gov/dluhc/printapi/rest/GetCertificateSummaryByApplicationIdIntegrationTest.kt:99]: Should the body be assserted?

**Review by Neil-Massey — APPROVED** [review]
LGTM - few minor suggestions

File: src/test/kotlin/uk/gov/dluhc/printapi/config/ApiRequestErrorAttributesTest.kt (hunk 1 above)
Comment (adacayi) [reply src/test/kotlin/uk/gov/dluhc/printapi/config/ApiRequestErrorAttributesTest.kt:75]: Thanks, @Neil-Massey, for looking into it in detail. 
I copied the error handling implementation from the VCA, so hopefully should be fine.

**Pull request merged**